frontend/src/app/components/wp-card-view/wp-single-card/wp-single-card.component.sass

```diff
@@ -40,10 +40,13 @@
   grid-template-columns: auto 1fr auto
   grid-template-rows: auto auto 1fr auto
   grid-row-gap: 10px
-  grid-template-areas: "type type type" "subject subject subject" "image image image" "attributeTag avatar idlink"
+  grid-template-areas: "project project project" "type type type" "subject subject subject" "image image image" "attributeTag avatar idlink"
   overflow: hidden
   flex-grow: 1
 
+  .wp-card--project-name
+    grid-area: project
+    font-style: italic
```

Comment:
You forgot to change the color to a light grey. Further when looking at it, I think we should also decrease the font-size to something like "12px".